### PR TITLE
fix: model style guide, views folder

### PR DESCRIPTION
**Pull request by vasilev-alex**

### CODE REVIEW

File: packages/cubejs-cli/src/templates.ts [hunk 1]
```diff
@@ -206,7 +206,7 @@ const ordersJs = `cube(\`Orders\`, {
   \`,
 
   preAggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-cli/src/templates.ts:209]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/src/scaffolding/formatters/BaseSchemaFormatter.ts [hunk 2]
```diff
@@ -118,10 +125,12 @@ export abstract class BaseSchemaFormatter {
     return {
       cube: tableSchema.cube,
       ...sqlOption,
+      ...dataSourceProp,
+      
       [this.options.snakeCase ? 'pre_aggregations' : 'preAggregations']: new ValueWithComments(
         null,
         [
-          'Pre-Aggregations definitions go here',
+          'Pre-Aggregation definitions go here',
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/src/scaffolding/formatters/BaseSchemaFormatter.ts:133]:
```suggestion
          'Pre-aggregation definitions go here',
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 3]
```diff
@@ -109,7 +109,7 @@ describe('ScaffoldingTemplate', () => {
   sql: \`SELECT * FROM public.orders\`,
   
   preAggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:112]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 4]
```diff
@@ -147,7 +147,7 @@ describe('ScaffoldingTemplate', () => {
   sql: \`SELECT * FROM public.customers\`,
   
   preAggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:150]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 5]
```diff
@@ -190,7 +190,7 @@ describe('ScaffoldingTemplate', () => {
   sql: \`SELECT * FROM public.accounts\`,
   
   preAggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:193]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 6]
```diff
@@ -250,7 +250,7 @@ describe('ScaffoldingTemplate', () => {
   sql_table: \`public.orders\`,
   
   pre_aggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:253]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 7]
```diff
@@ -288,7 +288,7 @@ describe('ScaffoldingTemplate', () => {
   sql_table: \`public.customers\`,
   
   pre_aggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:291]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 8]
```diff
@@ -331,7 +331,7 @@ describe('ScaffoldingTemplate', () => {
   sql_table: \`public.accounts\`,
   
   pre_aggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:334]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 9]
```diff
@@ -410,7 +410,7 @@ describe('ScaffoldingTemplate', () => {
   sql_table: \`public.\\\`someOrders\\\`\`,
   
   pre_aggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:413]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 10]
```diff
@@ -479,7 +479,7 @@ describe('ScaffoldingTemplate', () => {
   sql_table: \`public.orders\`,
   
   pre_aggregations: {
-    // Pre-Aggregations definitions go here
+    // Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:482]:
```suggestion
    // Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 11]
```diff
@@ -611,80 +611,101 @@ describe('ScaffoldingTemplate', () => {
           content: `cubes:
   - name: orders
     sql_table: public.orders
+
     pre_aggregations:
-      # Pre-Aggregations definitions go here
+      # Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:616]:
```suggestion
      # Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 12]
```diff
@@ -611,80 +611,101 @@ describe('ScaffoldingTemplate', () => {
...
         },
         {
           fileName: 'customers.yml',
           content: `cubes:
   - name: customers
     sql_table: public.customers
+
     pre_aggregations:
-      # Pre-Aggregations definitions go here
+      # Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:647]:
```suggestion
      # Pre-aggregation definitions go here
```

File: packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts [hunk 13]
```diff
@@ -611,80 +611,101 @@ describe('ScaffoldingTemplate', () => {
...
         },
         {
           fileName: 'accounts.yml',
           content: `cubes:
   - name: accounts
     sql_table: public.accounts
+
     pre_aggregations:
-      # Pre-Aggregations definitions go here
+      # Pre-Aggregation definitions go here
```
Comment (igorlukanin) [inline packages/cubejs-schema-compiler/test/unit/scaffolding-template.test.ts:682]:
```suggestion
      # Pre-aggregation definitions go here
```

File: packages/cubejs-api-gateway/src/gateway.ts [hunk 14]
```diff
@@ -303,10 +303,6 @@ class ApiGateway {
       });
     }));
 
-    /** **************************************************************
-     * Meta scope                                                    *
-     *************************************************************** */
-
```
[the remark this answers is not in the document]
Comment (ovr) [reply packages/cubejs-api-gateway/src/gateway.ts:309]: It's not a Turbo Pascal

**Review by ovr — APPROVED** [review]
LGTM

File: packages/cubejs-server-core/src/core/OptsHandler.ts [hunk 15]
```diff
@@ -501,21 +501,12 @@ export class OptsHandler {
...
       options.apiSecret = crypto.randomBytes(16).toString('hex');
       displayCLIWarning(
-        `Option apiSecret is required in dev mode. Cube.js has generated it as ${
+        `Option apiSecret is required in dev mode. Cube has generated it as ${
           options.apiSecret
         }`
       );
     }
 
-    // Create schema directory to protect error on new project with dev mode
```
Comment (ovr) [inline packages/cubejs-server-core/src/core/OptsHandler.ts:510]: Are you sure about this removal?

**Pull request merged**